package.json

```diff
@@ -13,7 +13,8 @@
   "homepage": "https://thelounge.github.io/",
   "scripts": {
     "start": "node index",
-    "build": "npm run build:grunt && npm run build:handlebars",
+    "build": "npm run build:font-awesome && npm run build:grunt && npm run build:handlebars",
+    "build:font-awesome": "mkdir -p client/fonts && cp node_modules/font-awesome/fonts/fontawesome-webfont.* client/fonts",
```

Comment:
> We're starting to abuse npm scripts.

Not at all, this is a very appropriate use of npm scripts. (Edit: Funny, Googling "npm scripts" bring "Why I Left Gulp and Grunt for npm Scripts" as first result :D)
For reference, I highly recommend reading http://blog.keithcirkel.co.uk/why-we-should-stop-using-grunt/ and http://blog.keithcirkel.co.uk/how-to-use-npm-as-a-build-tool/.

> Also not a particular fan of hardcoding the path like this.

I get you, but problem is, we have to somewhere. We even already do it [in our current Gruntfile](https://github.com/thelounge/lounge/blob/7157076aad1837db408b58c6b452bf20a8493b1a/Gruntfile.js#L15). I'm open to suggestions to make this better though.

> Yeah... Instead of copying files and running scripts, couldn't we make it so the server builds/serves all that stuff on demand?

I think building static assets on demand is overkill. Serving from demand also has its set of trade-offs, like the fact that the package needs to be installed by users as part of the `dependencies` (not saying it's good or bad, just an observation).

All in all, you both have good points for sure. My take on this is that something like this PR is still times better than shipping dependencies with the repo or going through Grunt with N new wrappers/plugins for it and their according DSLs.

@xPaw, @maxpoulin64, what do you guys say I fix this on Windows, we make this fetching-from-npm happen, then we improve the build mechanism? When I look at the sad fate of #62 solve-everything-at-once, I'd rather improve things step by step, or nothing will never happen. I'm not saying this solution is set in stone, but at least we can iterate on this.